aesthetic_ants/penguin/player.py

```diff
@@ -0,0 +1,30 @@
+import pyglet
+import pyglet.window.key as key
+from utils import angle_between, loader, keys
+from math import degrees
+
+
+class Player(pyglet.sprite.Sprite):
+    def __init__(self, x, y):
+        player_image = loader.image("penguin.png")
+
+        # Rotate about the center
+        player_image.anchor_x = player_image.width // 2
+        player_image.anchor_y = player_image.height // 2
+
+        super().__init__(player_image, x=x, y=y)
+
+    def update(self, **kwargs):
+        super().update(**kwargs)
+
+        if keys[key.W]:
+            self.y += 1
+        if keys[key.S]:
+            self.y -= 1
+        if keys[key.A]:
+            self.x -= 1
+        if keys[key.D]:
+            self.x += 1
+
```

Comment:
Subsequently this would have to also take `dt` into account as well.

`1` is a bit slow too, we should probably add a SPEED class var constant on Player that we can change and use that.